utils/array.js

```diff
@@ -18,6 +18,30 @@ function chunks(array, size = 1) {
   return result;
 }
 
+/**
+ * Checks if two arrays have any common items
+ * @param array1 {Array} - first array
+ * @param array2 {Array} - second array
+ * @returns {boolean} - true if the arrays have at least one common item, false otherwise
+ */
+
+function arraysHaveCommonItem(array1, array2) {
+  if (!Array.isArray(array1) || !Array.isArray(array2) || array1.length > 100 || array2.length > 100) {
```

Comment:
Why does this function care if the array lengths are more than a 100?

Reply:
I had added it for safety check but since we have a joi validator, this makes no sense, I will remove it.